sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.4.2 (2023-01-11)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Upgraded `azure-core` from `1.34.0` to version `1.35.0`.
```

Comment:
it's better also mention the azure-communication-common version upgrade

Reply:
Done